demo/MainWindow.axaml

```diff
@@ -27,55 +27,82 @@
         <Style Selector="Button i|Icon">
             <Setter Property="Margin" Value="0" />
         </Style>
+        <Style Selector="MenuItem i|Icon">
+            <Setter Property="FontSize" Value="{StaticResource FontSizeNormal}" />
+            <Setter Property="Margin" Value="0" />
+        </Style>
     </Window.Styles>
-    <Grid RowDefinitions="Auto,Auto" ColumnDefinitions="Auto,Auto" Margin="4">
-        <!--Font Awesome-->
-        <Grid RowDefinitions="Auto, Auto, Auto, Auto, Auto" ColumnDefinitions="Auto, Auto, Auto" Margin="4">
-            <TextBlock Text="FontAwesome" Grid.Row="0" Grid.ColumnSpan="2" FontSize="{StaticResource FontSizeLarge}" />
-            <!--GitHub-->
-            <TextBlock Text="fab fa-github" Grid.Row="1" />
-            <i:Icon Value="fab fa-github" Grid.Row="1" Foreground="BlueViolet" />
-            <Button i:Attached.Icon="fab fa-github" Grid.Row="1" Foreground="BlueViolet" />
 
-            <!--Address Card-->
-            <TextBlock Text="fas fa-address-card" Grid.Row="2" />
-            <i:Icon Value="fas fa-address-card" Grid.Row="2" />
-            <Button i:Attached.Icon="fas fa-address-card" Grid.Row="2" />
+    <NativeMenu.Menu>
+        <NativeMenu>
+            <NativeMenuItem Header="Open"/>
+            <NativeMenuItem Header="Recent">
+                <NativeMenuItem.Menu>
+                    <NativeMenu/>
+                </NativeMenuItem.Menu>
+            </NativeMenuItem>
+            <NativeMenuItem Header="Quit"  Gesture="CMD+Q"/>
+        </NativeMenu>
+    </NativeMenu.Menu>
```

Comment:
Should be removed as well.

Reply:
I didn't really test that because i have no mac, but if i recall once i lunch the program it crashed with the attached icon (Even on windows)